lib/Resolver.js

```diff
@@ -78,6 +78,19 @@ Resolver.prototype.resolve = function resolve(context, path, request, callback)
 	}, null));
 };
 
+Resolver.prototype.doResolveWithErrorLogging = function doResolveWithErrorLogging(type, request, message, callback, options) {
+	var optionsWithoutLog = {
+		stack: options.stack,
+		missing: options.missing
+	};
+
+	this.doResolve(type, request, message, createInnerCallback((err, result) => {
```

Comment:
My main concern was compatibility, it still needs to compare options.stack with callback.stack:

```
if(!log) {
		if(options.stack !== callback.stack) {
			var callbackWrapper = function callbackWrapper() {
				return callback.apply(this, arguments);
			};
			callbackWrapper.stack = options.stack;
			callbackWrapper.missing = options.missing;
			return callbackWrapper;
		}
		return callback;
	}
```